src/DelegationManagement.sol

```diff
@@ -10,7 +10,7 @@
  *
  */
 
-pragma solidity >=0.6.0 <=0.8.0;
+pragma solidity >=0.8.0;
```

Comment:
Please split the compiler change into a different commit on the same PR. Otherwise it is misleading to who is analyzing the PR

Reply:
sure